scrollytelling/package.json

```diff
@@ -17,19 +17,19 @@
     "react-dom": ">=18"
   },
   "dependencies": {
-    "@radix-ui/react-portal": ">=1",
-    "@radix-ui/react-slot": ">=1"
+    "@radix-ui/react-portal": "^1.0.4",
+    "@radix-ui/react-slot": "^1.0.2"
```

Comment:
let's not upgrade these — the `>=1` is intentional so that it has more chances of matching a version already installed by the user